src/nunit.analyzers.tests/ClassicModelAssertUsage/AreEqualClassicModelAssertUsageCodeFixTests.cs

```diff
@@ -295,15 +295,61 @@ public void Test(object actual, object expected)
         }
 
         [Test]
-        public void CodeFixPreservesLineBreakBeforeMessage()
+        public void CodeFixMaintainsReasonableTriviaWithEndOfLineClosingParen([Values] bool hasMessage)
+        {
+            var commaAndMessage = hasMessage
+                ? @",
+                ""message"""
+                : string.Empty;
```

Comment:
Not sure if the below is clearer or not.

```suggestion
            var commaAndMessage = hasMessage ? ",\n\"message\"" : string.Empty;
```

Reply:
@manfred-brands Yeah I didn't like this part of the code either.

So the spaces before the `""message""` do matter, so if we were to _not_ use a verbatim string, it should be:
```cs
            var commaAndMessage = hasMessage
                ? ",\r\n                \"message\""
                : string.Empty;
```
Do you find this more readable than the old code? Happy to make the change everywhere if you think so. (Note that the same applies to https://github.com/nunit/nunit.analyzers/pull/754#discussion_r1634236689)

Reply:
Both are not ideal, but my preference would be the latter. It clearly indicates a newline inside a string. From the previous one I missed the importance of the spaces.